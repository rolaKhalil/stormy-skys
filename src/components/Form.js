import React from 'react'

const Form = props => (
    <form onSubmit={props.getWeather}> 
        <input className="form-group" type='text' name='city' placeholder='City'/>
        <button>Weather</button>
    </form>   
)

export default Form