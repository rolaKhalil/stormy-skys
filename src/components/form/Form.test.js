import React from 'react'
import { shallow, configure } from 'enzyme'
import Form from './Form'
import renderer from 'react-test-renderer'

describe('<Form />', () => {
    it('renders without crashing', () => {
        const tree = renderer.create(
            <Form getWeather={'on submit function'} />
        )
        expect(tree.toJSON()).toMatchSnapshot()
    });
})