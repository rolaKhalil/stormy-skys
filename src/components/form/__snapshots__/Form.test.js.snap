// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> renders without crashing 1`] = `
<form
  onSubmit="on submit function"
>
  <input
    className="form-group"
    name="city"
    placeholder="City"
    type="text"
  />
  <button>
    Weather
  </button>
</form>
`;
